gapic/schema/wrappers.py

```diff
@@ -832,6 +832,11 @@ def has_lro(self) -> bool:
         """Return whether the service has a long-running method."""
         return any([m.lro for m in self.methods.values()])
 
+    @property
+    def has_pagers(self) -> bool:
+        """Return whether the service has paged methods."""
+        return any([m.paged_result_field for m in self.methods.values()])
+
```

Comment:
I'm not sure where/how to squeeze in the unit test for this to get coverage back to 100%.